Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -971,6 +970,41 @@ public class Dpad
             public string y;
         }
 
+
+        // RPC:
+        public enum RPCModes
+        {
+            ALL,
+            OTHERS,
+            HOST
+        }
+
+        [DllImport("__Internal")]
+        public extern static void RPCregister(string name);
+
+        [DllImport("__Internal")]
+        private extern static void RPCcallInternal(string name, string data, RPCModes mode, Action callbackOnResponse);
+
+        private static Action CallbackOnResponse = null;
```

Comment:
Have we tested this in the case of multiple RPC callback registrations? Wouldn't this be overridden in that case?

Reply:
> Have we tested this in the case of multiple RPC callback registrations? Wouldn't this be overridden in that case?

Just did a test with this and yes It does get overridden (Latest callback is called): 
![image](https://github.com/asadm/playroom-unity/assets/77355191/5314cdac-220c-4773-bf9a-e12760911101)

I guess holding a list of callbacks should work?

Reply:
Perhaps yes. Also, this brings up another edge case:
1. Multiple RPC callback registrations for the **same RPC event**
2. Multiple RPC callback registrations for **different RPC events**

Both should work.